docs/setup/components/register.md

```diff
@@ -0,0 +1,16 @@
+## Registration to layers
+
+The `component.register` tag registers the component to a specific layer.
```

Comment:
I think this wording is somewhat confusing because components can have multiple layers.